### DISCUSSION TITLE
problem di CLI

### DISCUSSION THREAD

**Discussion by mynodee** [deaafrizal/forum#26] [Troubleshooting Room]
selamat pagi siang atau malam, saya mau bertanya kira kira ini kenapa yah kok bisa error soalnya belum faham error nya kenapa terima kasih
![Screenshot 2023-12-10 090510](https://github.com/deaaprizal/forum/assets/138854270/5535c3e1-7e99-47c0-a809-fc19f1a1fddf)

**Comment by deaafrizal** [comment]
kita harus tau  dlu node js nya versi berapa bro. simpelnya coba uninstall node js, install versi 20 yg terbaru atau 18.

**Reply by mynodee** [reply]
node saya sekarang versi 18.17.1 kak, tapi ini mau aku coba upgrade dulu sih kak ke versi 20 an

**Comment by mynodee** [comment]
kak dea minta ternyata setelah node nya saya perbarui ternyata masih error mohon bantuan nya

**Reply by mynodee** [reply]
bang dea mau tanya kan ini sudah bisa gak error lagi tapi versi next.js nya versi 14 gak 13 kira masih bisa gas ikut project nya bang dea gak? terima kasih
